README.md

```diff
@@ -96,20 +102,25 @@ rpcuser=xxxxxx
 rpcpassword=xxxxxx
 
 rpcport=18232
-rpcallowip=0.0.0.0
+rpcallowip=127.0.0.1
 ```
+Reset your working directory to your `zingolib/reg/`
 
-In another terminal instance, navigate to your `lightwalletd` directory, and run
-`./lightwalletd --no-tls-very-insecure --zcash-conf-path ~/.zcash/zd/zcash.conf --data-dir ~/.zcash/zd/ --log-file ~/.zcash/ld/lwd-log --log-level 7`
+`zcashd` works in regtest with the following invocation (with explicit pathes - you will have to adjust to your own path):
+`./zcashd --printtoconsole -conf=/zingolib/.zc/zd/zcash.conf --datadir=/zingolib/.zc/zd/ -debuglogfile=/zingolib/.zc/zd/debug.log -debug=1`
```

Comment:
relative paths don't work?

Reply:
Yeah, not with `zcashd` flags, that I've seen. They mention it in the documentation, too.